bluezero/adapter.py

```diff
@@ -269,7 +269,8 @@ def _properties_changed(self, interface, changed, invalidated, path):
                 self.on_connect(new_dev)
             elif not changed['Connected'] and self.on_disconnect:
                 if tools.get_fn_parameters(self.on_disconnect) == 0:
-                    logger.warn("using deprecated version of disconnect callback , move to on_disconnect(dev) with device parameter")
+                    logger.warn("using deprecated version of disconnect callback , \\\
+                                 move to on_disconnect(dev) with device parameter")
```

Comment:
How you have split the line is valid but I prefer the following style:
```python
logger.warn("using deprecated version of disconnect callback , " +
            "move to on_disconnect(dev) with device parameter")
```

Reply:
ahah perfect ! I was looking for this script! sorry. we should be here now.

As I said if u will tag a version after your test I'll appreciate.

regards

Marco

Reply:
> ```python
> +
> ```

so do I , but it's slower ... of course it's not the case. By the way I changed it.